pkg/addons/addons_autopause.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors All rights reserved.
+Copyright 2022 The Kubernetes Authors All rights reserved.
```

Comment:
copyright stays same since it was created, no need to change this line

Reply:
Yes, will retain original copyright year.